### PR TITLE
Ignore some more problematic functions

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
They all need to be generated by hands.

cc @EPashkin @sdroege

### CODE REVIEW

**Comment by EPashkin** [comment]
@GuillaumeGomez Even after your gtk-rs/gir#667 ?
You will add regen or/and manual implementations in other PR?

**Comment by GuillaumeGomez** [comment]
The problem for those ones is that it has pointers of pointers. The problem is even before my PR. :)

**Comment by EPashkin** [comment]
Now reason is more clear, its preparing for time when closure implemented (as currently functions commented anyway).
Still comments `#array with size` is misleading for `request_targets` as no arrays in function definition, (only in its subtype),
also in `request_uris` type `ClipboardURIReceivedFunc` don't contains any size,

Still not clear for me if PR is ready for merge or not as it not pass "regen_check"

**Comment by GuillaumeGomez** [comment]
That's interesting: setting functions as ignored in this case make them generated (as commented but still). Surprising, no?

**Comment by EPashkin** [comment]
No, regen_check shows inverse diff, so these commented lines just removed

**Comment by GuillaumeGomez** [comment]
Is it expected?

**Comment by EPashkin** [comment]
Currently it shows changes that "wrongly added" to generated version.
Not fully sure is right or we better show just changes after "right done regen"

**Comment by GuillaumeGomez** [comment]
In any case it's a bit surprising since I just ignored some functions that aren't (yet) generated.

**Comment by EPashkin** [comment]
👍 , Travis again can't install clippy 😢

**Comment by GuillaumeGomez** [comment]
That's... interesting? XD Thanks for the review, I'll just merge then.

File: Gir.toml [hunk 1]
```diff
@@ -678,6 +678,22 @@ status = "generate"
     name = "set_can_store"
     #array with size
     ignore = true
+    [[object.function]]
+    name = "request_targets"
+    #array with size
```
Comment (sdroege) [inline Gir.toml]: What's the problem with this? We support that elsewhere just fine and we have conversion traits for this specific case :) `from_glib_none_num()`

Same for everything else below

Comment (GuillaumeGomez) [reply Gir.toml]: Then that means that my gir PR is incomplete. :'(

Comment (sdroege) [reply Gir.toml]: Indeed :(

**Pull request merged**